ynput/OpenPype
# Maya: Fix camera update

## Changelog Description
Fix resetting any modelPanel to a different camera when loading a camera and updating.

## Additional info
This is implementing fix from @BigRoy 🙏🏻 mentioned #2461.

## Testing notes:
1. Create scene in Maya, load camera in with OP
2. Update version of the loaded camera - panels should stay on that camera.

Resolves #2461
